src/appleseed/renderer/kernel/lighting/imagebasedlighting.cpp

```diff
@@ -228,8 +229,7 @@ void compute_ibl_environment_sampling(
 
         // Add the contribution of this sample to the illumination.
         env_value *= transmission / env_prob * mis_weight;
-        env_value *= material_value;
-        radiance += env_value;
+        madd(radiance, material_value, env_value);
```

Comment:
As you did just above, first multiply `env_value` by `transmission` to avoid spectrum copies.